lib/beaker/answers.rb

```diff
@@ -46,7 +80,20 @@ def self.create version, hosts, options
     # @param [String] default Should there be no user value for the provided question name return this default
     # @return [String] The answer value
     def answer_for(options, q, default = nil)
-      options[:answers][q] ? options[:answers][q] : default
+      answer = nil
```

Comment:
This can be `answer = DEFAULT_ANSWERS[q]`. If `DEFAULT_ANSWERS` does not contain `q`, the value will be `nil`. This would not impact setting the value to `default` later in the method.